README.md

```diff
@@ -28,7 +28,8 @@
 ![Czkawka](https://user-images.githubusercontent.com/41945903/145280350-506f7e94-4db0-4de7-a68d-6e7c26bbd2bf.gif)
 
 ## Supported OS
-Linux - Ubuntu 22.04+, Fedora 36+, Alpine Linux 3.16+, Debian 12+ and a lot of more   
+Linux - Ubuntu 22.04+, Fedora 36+, Alpine Linux 3.16+, Debian 12+ and a lot of more   ![image](https://user-images.githubusercontent.com/23202421/218338157-ece73ca7-489c-4204-9a6b-da9583063ae5.png)
```

Comment:
Hmm.......

Reply:
This Hmm... meant Hell No!  😄️

![Screenshot from 2023-02-20 07-33-00](https://user-images.githubusercontent.com/41945903/220030441-5e87a45b-0ab9-4132-9dff-0a8554d23578.png)

Reply:
ha! indeed this was not intended,

Reply:
> This Hmm... meant Hell No! smile
> 
> ![Screenshot from 2023-02-20 07-33-00](https://user-images.githubusercontent.com/41945903/220030441-5e87a45b-0ab9-4132-9dff-0a8554d23578.png)

I corrected the extra check, probably a miss click
and I added a few spaces, so the other neurotic doesn't have a heart attack :D